OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -126,17 +113,9 @@ public Actor[] SendParatroopers(Actor self, WPos target, bool randomize = true,
 			{
 				aircraftInRange[a] = false;
 
-				// Remove the camera when the final plane leaves the target area
+				// Remove the beacon when the final plane leaves the target area
 				if (!aircraftInRange.Any(kv => kv.Value))
 				{
-					if (camera != null)
-					{
-						camera.QueueActivity(new Wait(info.CameraRemoveDelay));
```

Comment:
Before this change the camera was only removed after the last plane had left the area. We have to update the remove delay to compensate for that.